prowler

```diff
@@ -320,6 +325,18 @@ if [[ -n "$ALLOWLIST_FILE" ]]; then
   allowlist
 fi
 
+# Check database providers
+if [[ ${DATABASE_PROVIDER} ]]
+then
+    if ! grep -q "${DATABASE_PROVIDER}" <<< "${SUPPORTED_DB_PROVIDERS}"
+    then
+        echo "Database provider: ${DATABASE_PROVIDER}"
+        echo "ERROR DB provider not supported - EXITING!"
+        echo "DATABASE PROVIDERS SUPPORTED: ${SUPPORTED_DB_PROVIDERS}"
+        exit 2
+    fi
+fi
+
```

Comment:
We need to check if `-M csv` is set because you need to generate the CSV prior to send it to DB, what do you think?